mappings/net/minecraft/predicate/entity/EntityPredicate.mapping

```diff
@@ -43,27 +43,20 @@ CLASS net/minecraft/class_2048 net/minecraft/predicate/entity/EntityPredicate
 		ARG 1 target
 	METHOD method_37227 (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;Lnet/minecraft/class_1297;)Z
 		ARG 3 entity
-	METHOD method_51704 toConditions (Lnet/minecraft/class_2048;)Lnet/minecraft/class_5258;
-		COMMENT Converts a single entity predicates into a predicate (loot condition) list.
-		COMMENT The returned list will have zero or one elements.
+	METHOD method_51704 asLootContextPredicate (Lnet/minecraft/class_2048;)Lnet/minecraft/class_5258;
 		ARG 0 predicate
-	METHOD method_51705 toConditions (Lcom/google/gson/JsonObject;Ljava/lang/String;Lnet/minecraft/class_5257;)Lnet/minecraft/class_5258;
-		COMMENT Deserializes a single entity predicate or a list of entity predicates in a JSON
-		COMMENT object's {@code key} field as a list of conditions.
-		ARG 0 obj
+	METHOD method_51705 fromJson (Lcom/google/gson/JsonObject;Ljava/lang/String;Lnet/minecraft/class_5257;)Lnet/minecraft/class_5258;
+		ARG 0 json
 		ARG 1 key
-		ARG 2 deserializer
-	METHOD method_51706 asConditions (Ljava/lang/String;Lnet/minecraft/class_5257;Lcom/google/gson/JsonElement;)Lnet/minecraft/class_5258;
+		ARG 2 predicateDeserializer
+	METHOD method_51706 fromJsonElement (Ljava/lang/String;Lnet/minecraft/class_5257;Lcom/google/gson/JsonElement;)Lnet/minecraft/class_5258;
 		ARG 0 key
-		ARG 1 deserializer
+		ARG 1 predicateDeserializer
 		ARG 2 json
-	METHOD method_51707 toConditonsArray (Lcom/google/gson/JsonObject;Ljava/lang/String;Lnet/minecraft/class_5257;)[Lnet/minecraft/class_5258;
-		COMMENT Deserializes a list of entity predicates in a JSON object's {@code key} array field,
-		COMMENT where each array element can be a single entity predicate or a list of entity
-		COMMENT predicates.
-		ARG 0 obj
+	METHOD method_51707 fromJsonArray (Lcom/google/gson/JsonObject;Ljava/lang/String;Lnet/minecraft/class_5257;)[Lnet/minecraft/class_5258;
```

Comment:
Notice the return type is `LootContextPredicate` instead of `EntityPredicate` for these methods. `fromXxx` sounds misleading in this case.